### PR TITLE
Created Vet-clinic DB

### PR DESCRIPTION
**Pull request by sonyco-4u**
- Created the Vet Clinic Database
- Created the animals table
- Copy the various SQL into the various sql files generated with the repo.
- Updated README.md
- Attached is a Screenshoots to the queries.

STEP 1 -2

STEP 3 -5

STEP 6 -8

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,48 +1,133 @@
...
+ </details>
+
+ <!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- An Animal table containing the following columns: Id, Name, Date of birth, Escape attempts, Neutered, Weight
+
+ <p align="right">(<a href="#readme-top">back to top</a>)</p>
+
```
Comment (omarsalem7) [inline README.md]: - [ ] Kindly fix the markdown format for key features.

File: README.md [hunk 2]
```diff
@@ -1,48 +1,133 @@
...
-Give a ⭐️ if you like this project!
+<!-- LICENSE -->
 
-## Acknowledgments
+## 📝 License <a name="license"></a>
 
-- Hat tip to anyone whose code was used
-- Inspiration
-- etc
+This project is [MIT](./LICENSE) licensed.
```
Comment (omarsalem7) [inline README.md]: - [ ] The link for a license is broken, kindly fix it and add your own license.

**Review by omarsalem7 — CHANGES_REQUESTED** [review]
Hi @sonyco-4u 😊, 

Good job so far! 👏
You have done well reaching this point of the milestone, but there are some issues that you still need to work on to go to the next project but you are almost there! 

you-are-near

## Highlights
- The Pull Request has a proper title and description. ✔️
- Correct git-flow ✔️

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
- N/A

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 2 above)
Comment (sonyco-4u) [reply README.md]: okay Noted

File: README.md (hunk 1 above)
Comment (sonyco-4u) [reply README.md]: Done

File: README.md (hunk 2 above)
Comment (sonyco-4u) [reply README.md]: I have made the corrections, it's working perfectly.

**Review by Reem-lab — APPROVED** [review]
# Hi @sonyco-4u, 👋🏻👋🏻 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉🚀🔥🎉🎉 


## Highlights ✨✨
- Followed Gitflow ✅
- Great working on the functionality 🔥🚀🎉
- Descriptive PR ✅


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**